NUService.js

```diff
@@ -156,6 +156,22 @@ export default class NUService extends NUObject {
         return url;
     }
 
+    buildBulkURL(entities, RESTResourceName, parentEntity) {
```

Comment:
Please confirm is bulkDelete supports parent-child related urls. Most likely it is directly on the entity.
e.g. /domains/id=A

Reply:
removed